scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -716,14 +720,7 @@ class ScalametaParser(input: Input)(implicit dialect: Dialect) { parser =>
       }
 
       val openParenPos = in.tokenPos
-      accept[LeftParen]
-      val ts = if (!token.is[RightParen]) listBy[Type] { tsBuf =>
-        do {
-          tsBuf += paramOrType()
-        } while (acceptOpt[Comma] || token.is[Ellipsis])
-      }
-      else Nil
-      accept[RightParen]
+      val ts = inParensOr(commaSeparated(paramOrType()))(Nil)
```

Comment:
```suggestion
      val ts = inParensOr(commaSeparated(paramOrType()))(syntaxError(...))
```
? since we expect them and we can give a meaningful error message here? Parens are expected and if there are not there we should show error.